packages/nx/src/project-graph/plugins/internal-api.ts

```diff
@@ -107,8 +108,9 @@ export async function loadNxPlugins(
       ? loadNxPluginInIsolation
       : loadNxPlugin;
 
-  plugins = await normalizePlugins(plugins, root);
-
+  if (!skipDefaultPlugins) {
+    plugins = await normalizePlugins(plugins, root);
+  }
```

Comment:
We still need to normalize to an empty array at minimum, that could either be done in a ternary here or by passing the flag through to normalize plugins, either way.
```suggestion
    plugins =  skipDefaultPlugins ? (plugins ?? []) : await normalizePlugins(plugins, root);
```